setup.cfg

```diff
@@ -0,0 +1,3 @@
+[flake8]
+ignore = E402,W504
+max-line-length = 160
```

Comment:
Let's add the comment that this is the policy used by ansible, that's why we use it as well.

Reply:
Are there any other guidelines we should have it here?

Reply:
Nope, ansible only has this in its [tox.ini](https://github.com/ansible/ansible/blob/devel/tox.ini#L26).

Reply:
We should get rid of W504

Reply:
No, not really: https://github.com/ansible/ansible/blob/devel/test/sanity/pep8/current-ignore.txt

Reply:
And before anyone asks, below is a **tiny, tiny** excerpt from running flake8 against current ansible upstream. Bottom line: I really don't see any point in us trying to enforce stricter coding standards than upstream has.

```./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:576:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:577:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:578:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:579:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:580:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:581:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:582:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:583:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:584:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:585:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:586:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:587:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:591:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:597:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_cluster.py:601:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_nic.py:173:17: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_nic.py:174:17: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_nic.py:175:17: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_nic.py:180:17: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_nic.py:181:17: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_host.py:305:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_host.py:306:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_host.py:307:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_host.py:308:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_host.py:309:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_host.py:310:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_host_pm.py:166:13: W504 line break after binary operator
./lib/ansible/modules/cloud/ovirt/ovirt_host_pm.py:167:13: W504 line break after binary operator```

Reply:
Actually I've change to fix W504 and Ansible maintainers changed back, no idea why, I think `line break after binary operator` shouldn't really be, but they seems to prefer it.